README.md

```diff
@@ -1 +1,57 @@
-# puppet-profile_mysql_server
\ No newline at end of file
+# profile_mysql_server
+
+![pdk-validate](https://github.com/ncsa/puppet-profile_mysql_server/workflows/pdk-validate/badge.svg)
+![yamllint](https://github.com/ncsa/puppet-profile_mysql_server/workflows/yamllint/badge.svg)
+
+NCSA Common Puppet Profiles - install and configure MySQL/MariaDB Server
+
+Oriented towards MariaDB.
+
+Intends to offer support for stateful or stateless nodes, but generally with persistent storage for the DB.
+
+
+## Table of Contents
+
+1. [Description](#description)
+1. [Setup](#setup)
+1. [Usage](#usage)
+1. [Dependencies](#dependencies)
+1. [Reference](#reference)
+
+
+## Description
+
+This puppet profile installs and configures MySQL/MariaDB Server on a node.
```

Comment:
I would clarify that this sets parameters for the `puppet/mysql` module, which does the actual install.

Reply:
Done.